stores/visible.ts

```diff
@@ -33,11 +33,12 @@ export let visible = computed<VisibleValue, [typeof current, typeof support]>(
     } else {
       let fallback = formatRgb(toRgb(color))
       if (inP3(color)) {
-        return {
+        let result: VisibleValue = {
```

Comment:
Why we need this change?

Reply:
I reverted the change in this commit: https://github.com/evilmartians/oklch-picker/pull/35/commits/5dd3f2420e158aad96e7a4e670202cd4327f7be3